src/sentry/notifications/notification_action/utils.py

```diff
@@ -26,8 +26,13 @@ def should_fire_workflow_actions(org: Organization, type_id: int) -> bool:
             and features.has("organizations:workflow-engine-single-process-workflows", org)
         )
         or features.has(
-            "organizations:workflow-engine-trigger-actions", org
-        )  # This is for temporary rollouts
+            "organizations:workflow-engine-single-process-metric-issues",
+            org,  # Metric issue single processing
```

Comment:
I think this should check the group type so orgs on this flag will not suddenly have all groups firing issue alerts through the new system